atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/sweep/BackgroundSweeperImpl.java

```diff
@@ -332,8 +337,11 @@ private void saveSweepResults(final SweepProgressRowResult progress,
         if (cellsDeleted > 0) {
             Stopwatch watch = Stopwatch.createStarted();
             kvs.compactInternally(TableReference.createUnsafe(progress.getFullTableName()));
+            long elapsedMillis = watch.elapsed(TimeUnit.MILLISECONDS);
             log.debug("Finished performing compactInternally on {} in {} ms.",
-                    progress.getFullTableName(), watch.elapsed(TimeUnit.MILLISECONDS));
+                    progress.getFullTableName(), elapsedMillis);
+            sweepPerfLogger.logInternalCompaction(progress.getFullTableName(), cellsDeleted,
+                    cellsExamined, elapsedMillis);
```

Comment:
I'm going to push back on this, because currently this PR increases tech debt (these related variables are passed together into a different class with an API that you're just introducing). It will be harder to refactor once this goes in.

Given the time difference and given that the change should be fairly quick, I might just do this myself this afternoon.